tools/features/com.liferay.ide.hidden/feature.xml

```diff
@@ -31,6 +31,10 @@
    </license>
 
    <includes
+         id="com.ianbrandt.tools.m2e.mdp.feature"
```

Comment:
Do we need this?  This should be handled just like the dependencies for m2e-wtp and m2e itself which aren't in this file